index.js

```diff
@@ -378,11 +378,13 @@ function Argv (processArgs, cwd) {
   }
 
   var versionOpt = null
-  self.version = function (ver, opt, msg) {
-    versionOpt = opt || 'version'
-    usage.version(ver)
+  self.version = function (opt, msg, ver) {
+    var version = ver || msg || opt || undefined
```

Comment:
I think we can simply this logic a bit, and avoid the ternary operator. I might be tempted to do something like this:

1. check the # of arguments, if there's only **1** assume that it represents the version #.
2. if there are **2** assume that it's `opt`, `version`.
3. if there are **3** assume `opt`, `description`, `version`.



```js
self.version = function (opt, desc, version) {
  if (arguments.length === 1) {
    version = opt
    opt = 'version'
  } else if (arguments.length === 2) {
    version = desc
    desc = null
  }

  desc = desc || usage.deferY18nLookup('Show version number')
  usage.version(version)
  self.boolean(opt)
  self.sescribe(opt, desc)
}
```